README.md

```diff
@@ -119,8 +119,21 @@ xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
 			          
 *reghdfe and two-way clustering
 xtevent ln_w age c.age#c.age ttl_exp c.ttl_exp#c.ttl_exp tenure , ///
-            pol(union) w(3) cluster(idcode year) reghdfe ///
-			proxy(wks_work)
+            pol(union) w(3) impute(nuchange) cluster(idcode year) reghdfe ///
+            proxy(wks_work)
+            
+*Interaction Weighted Estimator

```

Comment:
@Constantino-Carreto-Romero This is a bit unclear, shouldn't we quote this as the Sun and Abraham estimator?

Reply:
I edited it in https://github.com/JMSLab/xtevent/commit/602cf1d6bbdbb4083f9cba70c6b0d47b3f158c51